src/lib/user/user-actor.ts

```diff
@@ -124,11 +127,25 @@ export class UserActor {
   }
 
   // TODO: Test UserActor::getDogReports
```

Comment:
See if we can do this.

Reply:
The plan is to test this when VetActor can create the reports. That way we only need to mock user and vet actors and use those to write the tests.

Reply:
TODO item updated in 5f69d730

Reply:
Is the behaviour covered by UI Tests at least? If so add to the comment—yes or no—and if yes which test.

Reply:
Yes it is covered by the "user can view report and edit sub-profile" UI Test.

TODO item updated in 76520e09